linode/manager
# Refactor: [M3-6280] MUI Migration: Profile

## Description 📝
MUI migration for the whole Profile section

The PR seems large but only features MUI migrations & code cleanup (exports, ordering, prop names). Managed to do it all with styled components and occasional `sx` props, moving us away from makeStyles()

## How to test 🧪
1. navigate /profile
2. go through all flows, compare with production styles
3. confirm no regression was introduced as part of this PR
